packages/wonder-blocks-search-field/src/components/search-field.js

```diff
@@ -213,3 +222,17 @@ const styles = StyleSheet.create({
         paddingRight: Spacing.large_24 + Spacing.medium_16,
     },
 });
+
+type ExportProps = $Diff<
+    React.ElementConfig<typeof SearchField>,
+    WithForwardRef,
+>;
+
+const ForwardedSearchField: React.AbstractComponent<
+    ExportProps,
+    HTMLInputElement,
+> = React.forwardRef<ExportProps, HTMLInputElement>((props, ref) => (
+    <SearchField {...props} forwardedRef={ref} />
+));
```

Comment:
We could just implement the component inside of the callback.  This would avoid the need to use `forwardedRef` and differentiate between `Props` and `ExportProps`.  I don't think we do this anywhere yet, check with @jandrade first and see what he thinks.

Reply:
ohhh yeah good point! that should work fine with this component. I'm happy that it was refactored to a functional one b/c it makes easier this change.